File: go/4_6/4_6_0.go
// go build 4_6_0.go && ./4_6_0
package main

import (
	"fmt"
	"time"
)

func main() {
	start := time.Now()

	i := 0
	var sum int64 = 0
	for i < 1000000 {
		//fmt.Println(i)
		sum += int64(i)
		i++
	}

	duration := time.Since(start)
	fmt.Printf("合計: %d\n", sum)
	fmt.Printf("実行時間: %f秒\n", duration.Seconds())
}

File: c/4_6_0.c
//cc 4_6_0.c && ./a.out
//cc -O2 4_6_0.c && ./a.out
#include <stdio.h>
#include <time.h>

int main() {
    clock_t start = clock();

    int i = 0;
    long long sum = 0;
    while (i < 1000000) {
        //printf("%d\n", i);
        sum += i;
        ++i;
    }

    clock_t end = clock();

    double duration = (double)(end - start) / CLOCKS_PER_SEC;
    printf("合計: %lld\n", sum);
    printf("実行時間: %f秒\n",duration);

    return 0;
}